x-pack/test_serverless/functional/test_suites/observability/cases/list_view.ts

```diff
@@ -258,6 +259,7 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
         createNCasesBeforeDeleteAllAfter(1, getPageObject, getService);
 
         it('deletes a case correctly', async () => {
+          await toasts.dismissAll();
```

Comment:
I am wondering if we should do it before each test. Wdyt?

Reply:
If you'd like to, please update this branch, so we can git it in quicker.  
I don't see the harm in it, personally.

Reply:
If we are in a rush to merge this PR then it is better to leave it as it is.